cabal-install/src/Distribution/Client/VCS.hs

```diff
@@ -305,7 +314,43 @@ vcsDarcs =
 
     vcsSyncRepos :: Verbosity -> ConfiguredProgram
                  -> [(SourceRepositoryPackage f, FilePath)] -> IO [MonitorFilePath]
-    vcsSyncRepos _v _p _rs = fail "sync repo not yet supported for darcs"
+    vcsSyncRepos verbosity prog = \ case
```

Comment:
LambdaCase is almost absent from the cabal codebase, I think we can avoid it here too and just match directly on rs instead, since it's just two cases.

Reply:
Done